### ISSUE TITLE
Adding memory

### ISSUE THREAD

**Issue by simison** [Trustroots/trustroots#166]
I'll double up the memory from 1GB to 2GB.

Also moving VPS from London to Amsterdam, DigitalOcean seems to have better scalability possibilities there and it's closer to most of the registered users anyway.

~30min downtime now due all this. Up probably before 2:00 AM CET.

**Comment by simison** [comment]
* IP change from 178.62.110.65 to 188.166.32.52
* Enabled IPv6

Ping @chmac @guaka you guys might need to remove old trustroots.org from your ~/.ssh/known_hosts file.

**Comment by simison** [comment]
Site is up and running again. Old VPS is proxying to the new one:
```
location / {
        proxy_pass https://188.166.32.52;
        proxy_redirect          off;
        proxy_set_header        Host            $host;
        proxy_set_header        X-Real-IP       $remote_addr;
        proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
}
```

**Comment by guaka** [comment]
lower TTL next time: #167

**Comment by simison** [comment]
Hmm, looks like there's something funny with old VPS proxy not working now, I'll check that.

**Comment by guaka** [comment]
could it be that console is getting equivalent of 
![screen shot 2015-01-07 at 11 19 41](https://cloud.githubusercontent.com/assets/54728/5644014/1a28d14c-965f-11e4-8f2e-b35981c44abf.png)

**Comment by simison** [comment]
Fixed proxy from old 178.62.110.65, something with ssl_ciphers was interrupting everything else there.

Now even tho people end up to the old one, they'll see the new server just fine. Test by editing your hosts file:
```
178.62.110.65 trustroots.org
178.62.110.65 www.trustroots.org
```
